test/AutoDiff/stdlib/tgmath_derivatives.swift.gyb

```diff
@@ -1,12 +1,22 @@
 // RUN: %target-run-simple-swiftgyb(-Xfrontend -enable-experimental-forward-mode-differentiation)
 // REQUIRES: executable_test
 
+// TODO(philipturner) Once SwiftCompiler/Sources/ExperimentalRegex isn't broken, 
+//   try removing the explicit overlay imports and pass `-Xfrontend 
+//   -enable-cross-import-overlays` to this test's "RUN"
+
 #if canImport(Darwin)
-  import Darwin.C.tgmath
+  import Darwin
+  import _Darwin_Differentiation
 #elseif canImport(Glibc)
   import Glibc
+  import _Glibc_Differentiation
+#elseif os(WASI)
+  import WASILibc
+  import _WASILibc_Differentiation
 #elseif os(Windows)
   import CRT
+  import _CRT_Differentiation
```

Comment:
We cannot break existing clients and require them to explicitly import overlays.

Reply:
Same reason as above. Should this PR be converted into a draft until the build failure is fixed?

Reply:
It should be a draft as this PR is not ready.